cypress/support/commands.js

```diff
@@ -24,6 +24,29 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 // one of the fec dependencies talks to window.insights.chrome
+import React, { Suspense } from 'react';
+import { Provider } from 'react-redux';
+import { init } from '../../src/Store';
+import messages from '../../locales/data.json';
+import IntlProvider from '@redhat-cloud-services/frontend-components-translations/Provider';
+import { mount } from '@cypress/react';
+
+Cypress.Commands.add('mountWithContext', (Component, FallBackComponent) => {
+
+    return mount(
+        <IntlProvider messages={messages}>
+            <Provider store={ init().getStore() }>
+                {FallBackComponent ?
+                    (<Suspense fallback={FallBackComponent}>
+                        <Component />
+                    </Suspense>) :
```

Comment:
What's the reason for explicit Suspense support here? You could use the Component argument and pass anything you want inside of it including components wrapped in Suspense. This would help make the function more generic...